index.html

```diff
@@ -0,0 +1,198 @@
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="wrapper"><a href="#">See project</a></button>
+                </article>
+            </article>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <button type="button" class="wrapper"><a href="#">Get My Resume</a></button>
+        </div>
+        <div class="about-me-skils">
+            <div class="worklanguages">
+                <img src="./images/icon-languages.svg" alt="Image depicting languages">
+                <h3>Languages</h3>
+                <div class="lang-tags">
+                    <ul>
+                       <li>HTML</li>
+                       <li>Ruby</li>
+                       <li>JavaScript</li>
+                       <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="frameworks">
+                <img src="./images/icon-frameworks.svg" alt="Image that represent frameworks">
+                <h3>Frameworks</h3>
+                <div class="frameworks-tags">
+                    <ul>
+                       <li>Bootstrap</li>
+                       <li>Ruby On Rails</li>
+                       <li>RSpec</li>
+                       <li>Copybara</li>
+                       <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="skills">
+                <img src="./images/icon-skills.svg" alt="An image that shows the skills">
+                <h3>Skills</h3>
+                <div class="skills-tags">
+                    <ul>
+                       <li>Database Management</li>
+                       <li>Version Control</li>
+                       <li>CLI</li>
+                       <li>Web Development</li>
+                       <li>API Design</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="contact-me">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/xvojoboz" method="POST" class="contactform">
+           <input required type="text" id="fname" name="fname" placeholder="Full name" maxlength="30"><br><br>
+           <input required pattern="^([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$" type="text" id="email" name="email" placeholder="Email address"><br><br>
+           <textarea required name="message" maxlength="500">Hey I'd like to connect to...</textarea><br><br>
```

Comment:
- [ ] [Optional] Nice work so far 👍🏿 I would suggest you use a placeholder for the text instead of hard coding the text in the textarea